components/normalizer/benches/canonical_decomposition.rs

```diff
@@ -13,22 +13,7 @@ fn function_under_bench(canonical_decomposer: &CanonicalDecomposition, character
 pub fn criterion_benchmark(criterion: &mut Criterion) {
     let group_name = "canonical_decomposition";
     let decomposer = CanonicalDecomposition::try_new_unstable(&icu_testdata::unstable()).unwrap();
-    let params = black_box([
-        'ä',
-        'Ä',
-        'ệ',
-        'Ệ',
-        '\u{1D15E}',
-        'ো',
-        '𑄮',
-        '가',
-        '각',
-        '\u{212B}',
-        '\u{2126}',
-        '\u{1F71}',
-        '\u{1F72}',
-        'ά',
-    ]);
+    let params = black_box([ 'a', 'ä', 'Ä' ]);
```

Comment:
These data still look synthetic. My expectation for a collator test is that we test real-life usages, not synthetic test data. For example, we could try to sort language display names:

https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-localenames-full/main/fo/languages.json

CC @markusicu @echeran

Reply:
This is for normalization rather than collation. That list of strings doesn't seem to exercise normalization in a sufficiently benchmark-relevant way.

Reply:
I've used the test data from the files in /tests, but the commited file is almost empty. (need to download manully to run it).

Is it going to be an issue for benching or not ?

Reply:
I added first part of the file just for the compilation to go ahead.

Reply:
resolving this.

The test file (at least when running on CI jobs), is the start of the file. 

to run a full bench run on the file locally, the file will need to be downloaded manually, unless we decide to push the file in github.